yarn.lock

```diff
@@ -11087,7 +11092,7 @@ react-docgen@^5.0.0:
     node-dir "^0.1.10"
     strip-indent "^3.0.0"
 
-react-dom@^16.9.11:
+react-dom@16.14.0, react-dom@^16.9.11:
```

Comment:
the devDependency versions are different for `"@types/react-dom": "^16.9.11"` and `"react-dom": "16.14.0"`. Could you update both to the same version "16.14.0"?

Reply:
16.14.0 unfortunately does not exist, so we will keep the original version